.env.example

```
@@ -62,6 +62,14 @@ HARMONIA_GROQ_API_KEY=""
 # Secret for Vercel cron to trigger organize.run
 HARMONIA_CRON_SECRET=""
 
+# -----------------------------------------------------------------------------
+# BILLING (Polar.sh)
+# -----------------------------------------------------------------------------
+POLAR_ACCESS_TOKEN=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)